src/components/Checkbox.js

```diff
@@ -8,10 +8,41 @@ import { useOpenWebScreen } from '../hooks';
 import { OrientationContext } from '../OrientationProvider';
 
 import { BoldText, RegularText } from './Text';
-import { WrapperHorizontal, WrapperRow } from './Wrapper';
+import { WrapperHorizontal } from './Wrapper';
 
 const { a11yLabel } = consts;
 
+const renderTitleContent = ({
+  boldTitle,
+  lightest,
+  link,
+  linkDescription,
+  navigate,
+  openWebScreen,
+  title
+}) => {
+  const TextComponent = boldTitle ? BoldText : RegularText;
+
+  return (
+    <WrapperHorizontal>
+      <TextComponent small lightest={lightest}>
+        {title}
+        {(!!link || !!navigate) && !!linkDescription && (
+          <RegularText
+            small
+            primary
+            underline
+            onPress={link ? openWebScreen : navigate}
+            style={{ flexShrink: 1 }}
+          >
+            {linkDescription}
+          </RegularText>
+        )}
+      </TextComponent>
```

Comment:
this will nest a text component inside a text component. i think there can be issues on some operating systems? this was not the case before, how come this is changed?

Reply:
Previously, two text components were wrapped with `WrapperRow`. This was causing the text in the `Checkbox` to jump off the screen. In this way, it is ensured that it descends to the bottom line without going off the screen.
Maybe we should use wrapping with `WrapperWrap`?

|before|after|after (with WrapperWrap)|
|--|--|--|
![Simulator Screenshot - iPhone 16 Plus - 2025-05-09 at 11 52 38](https://github.com/user-attachments/assets/b87946f1-5f84-4c1e-974a-76c55054a86a)|![Simulator Screenshot - iPhone 16 Plus - 2025-05-09 at 11 53 05](https://github.com/user-attachments/assets/17654d1c-ae78-43d4-8efa-a3965aa9635b)|![Simulator Screenshot - iPhone 16 Plus - 2025-05-09 at 11 52 54](https://github.com/user-attachments/assets/8d0a96f9-32c4-4fbc-bfe3-fc533c0bf2cb)

Reply:
if it works well for all platforms, we can keep the text nesting.

Reply:
Yes, I tested it for both platforms and did not encounter any problems.